frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -1717,7 +1769,15 @@ const ManageHostsPage = ({
             !showSelectedLabel &&
             !mdmId &&
             !mdmEnrollmentStatus &&
+            !munkiIssueId &&
             renderOSFilterBlock()}
+          {!!munkiIssueId &&
```

Comment:
Im starting to thing we need to pull out the activity block into its own component. this keeps getting longer the more filter options we have. I dont think we have to do it in this PR but if we add another we should look at moving it out

Reply:
Yeah, it's really adding to ManageHostPage.tsx now. I'll start a new ticket for this.

Reply:
https://github.com/fleetdm/fleet/issues/7558

Reply:
There's probably an opportunity to cut down on the conditionals with something like `reconcileExclusiveHostParams`.